### PR TITLE
Service provider creates workshop with competitive selection without

### PR DESCRIPTION
**Pull request by SergeyGSA**
Added:
- the competitive selection radio buttons and description textarea
- the competitive selection section to workshop-card and workshop-details components
- the hint with competitive selection description in the workshop-details component

The link on the story:
https://github.com/ita-social-projects/OoS-Backend/issues/214

### CODE REVIEW

File: src/app/shell/details/workshop-details/workshop-details.component.html [hunk 1]
```diff
@@ -21,9 +21,23 @@ <h1 class="title">{{ workshop.title }}</h1>
...
       <div fxLayout="row" fxLayoutAlign="start center">
         <a [routerLink]="['/details/provider', workshop.providerId]">
-          <span class="icon-text details-link"> <mat-icon class="icon">home</mat-icon>{{ workshop.providerTitle }} </span>
+          <span class="icon-text details-link" [ngClass]="{'workshop-title': workshop.competitiveSelection}"> <mat-icon class="icon">home</mat-icon>{{ workshop.providerTitle }} </span>
         </a>
       </div>
+      <div fxLayout="row" fxLayoutAlign="start center" class="selection" *ngIf="workshop.competitiveSelection"> 
+        <img class="icon icon-dimensions selection-icon" src="../../../../assets/icons/competitiveSelection.svg">
+        <span class="icon-text selection-text">{{ 'COMPETITIVE_SELECTION' | translate }}</span>
+        <i
```
Comment (humanmantis) [inline src/app/shell/details/workshop-details/workshop-details.component.html]: fix formatting for `i` tag. Always keep closed brackets on same line

File: src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/create-about-form.component.html [hunk 2]
```diff
@@ -208,18 +208,26 @@
     </mat-radio-group>
   </div>
 
-  <!-- <label class="step-label"> Конкурсний відбір </label>
-  <mat-radio-group [formControl]="competitiveSelectionRadioBtn" color="primary" fxLayout="column"
-    fxLayoutAlign="center space-between">
+  <label class="step-label"> Конкурсний відбір </label>
+  <mat-radio-group [formControl]="competitiveSelectionRadioBtn" color="primary" fxLayout='column'
+    fxLayoutAlign='center space-between'>
```
Comment (humanmantis) [inline src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/create-about-form.component.html]: use double quotes in template

File: src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/create-about-form.component.ts [hunk 3]
```diff
@@ -61,6 +60,8 @@ export class CreateAboutFormComponent implements OnInit, OnDestroy {
     this.PassAboutFormGroup.emit(this.AboutFormGroup);
     this.workshop && this.activateEditMode();
     this.initListeners();
+    this.onCompetitiveSelectionCtrlInit();
```
Comment (humanmantis) [inline src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/create-about-form.component.ts]: can be moved to `initListeners` and renamed to match the others.

**Review by humanmantis — CHANGES_REQUESTED** [review]

**Comment by SergeyGSA** [comment]
I added the last commit because when I, as SP, tried to create a new workshop, I noticed the error "cannot read property of null". So I added the FormControl with an empty string by default instead of null

**Review by humanmantis — APPROVED** [review]

**Pull request merged**